packages/in_app_purchase/in_app_purchase_android/lib/src/billing_client_wrappers/billing_client_wrapper.dart

```diff
@@ -82,14 +80,19 @@ class BillingClient {
 
   /// Enable the [BillingClientWrapper] to handle pending purchases.
   ///
+  /// **Deprecation warning:** it is no longer required to call
+  /// [enablePendingPurchases] when initializing your application.
+  ///
   /// Play requires that you call this method when initializing your application.
   /// It is to acknowledge your application has been updated to support pending purchases.
   /// See [Support pending transactions](https://developer.android.com/google/play/billing/billing_library_overview#pending)
   /// for more details.
   ///
   /// Failure to call this method before any other method in the [startConnection] will throw an exception.
```

Comment:
This should be removed; it is no longer true.

Reply:
Removed.